packages/devtools/lib/src/trees.dart

```diff
@@ -74,7 +72,7 @@ class TreeNode<T extends TreeNode<T>> {
     child.index = children.length - 1;
   }
 
-  bool containsChildWithCondition(Condition condition, {T root}) {
+  bool containsChildWithCondition(bool condition(T), {T root}) {
```

Comment:
this needs to be
`bool condition(T node)`
right now Dart thinks the type of the condition parameter arg is dynamic.

Reply:
whoops. fixed.